exec/java-exec/src/test/java/org/apache/drill/test/ClusterFixture.java

```diff
@@ -57,10 +40,27 @@
 import org.apache.drill.exec.store.mock.MockStorageEngineConfig;
 import org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider;
 import org.apache.drill.exec.util.StoragePluginTestUtils;
-
 import org.apache.drill.shaded.guava.com.google.common.base.Charsets;
 import org.apache.drill.shaded.guava.com.google.common.base.Preconditions;
+import org.apache.drill.shaded.guava.com.google.common.collect.ImmutableMap;
 import org.apache.drill.shaded.guava.com.google.common.io.Resources;
+import org.apache.drill.test.DrillTestWrapper.TestServices;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
```

Comment:
Can we define a Drill standard for import order? The Apache settings I imported into Eclipse are `java`, `javax`, `org`, `com`. Statics are at the top.

In this change, looks like `java` moves the to bottom.

Reply:
I use standard IntelliJ IDEA imports order:

<img width="1331" alt="image" src="https://user-images.githubusercontent.com/15086720/67193155-811dcc80-f3fd-11e9-8434-53b9a5f598cf.png">

Regarding updating the check styles, there is a Jira (https://issues.apache.org/jira/browse/DRILL-7352) where we can post comments.